server/command_create.go

```diff
@@ -14,15 +14,30 @@ import (
 
 var installationNameMatcher = regexp.MustCompile(`^[a-zA-Z0-9-]*$`)
 
+const (
+	// Command args
+	size          = "size"
+	version       = "version"
+	affinity      = "affinity"
+	license       = "license"
+	filestore     = "filestore"
+	database      = "database"
+	enterpriseTag = "enterprise-tag"
+	testData      = "test-data"
+
+	enterpriseRepo = "mattermost/mattermost-enterprise-edition"
+)
+
```

Comment:
I like this change to use `const`s for these values and have honestly wondered why we hardcode these values in several repos

Reply:
I think for speed when prototyping, and later we forget 😄